src/components/textarea.js

```diff
@@ -36,6 +36,9 @@ module.exports = function(app) {
             }
             return 'formio/components/texteditor.html';
           }
+          else if (!$scope.readOnly && $scope.component.components) {
```

Comment:
Hi Travis,

Was aware that container components contained components.  Just hadn't thought about the confusion caused when applied to a textarea.  Will be happy to rename to 'editorComponents'.  Will also affect related pull request in formio repository.